provider/resources.go

```diff
@@ -7004,5 +7004,21 @@ func Provider() tfbridge.ProviderInfo {
 	// Add a CSharp-specific override for aws_s3_bucket.bucket.
 	prov.Resources["aws_s3_bucket_legacy"].Fields["bucket"].CSharpName = "BucketName"
 
+	prov.P.ResourcesMap().Range(func(key string, value shim.Resource) bool {
```

Comment:
Do we expect this to apply to all fields called `"tags_all"`? If so, I would love a comment to explain that `"tags_all" is commonly used to inherit tags, and so ...`. If not, can we narrow this to only apply to the closed set of resources where this is a problem.

Reply:
Updated this to match the upstream provider, which checks for a computed tags_all and a non-computed tags: https://github.com/pulumi/pulumi-aws/blob/a82c36c5556e2b8de91ba2a82a24bcd6074345be/internal/provider/provider.go#L394-L415